doc/feature/morphology11.md

```diff
@@ -0,0 +1,128 @@
+Implementing Morphology version 1.1, support for glia cells
+============
+
+The current morphology version 1 specification and implementation is tailored
+for neuronal cells. The support for glia does imply only some changes and will
+be backward compatible to version 1, hence version 1.1.
+
+## Requirements
+
+* Backward-compatibility with version 1
+* Support for glia cells according to
+  [specification](https://bbpcode.epfl.ch/code/#/c/22473/20/source/h5v1.rst):
+  * new 'cell_family' HDF5 enum which has the one of the values NEURON or GLIA
+  * new glia cell values for brion::SectionType: 'glia process', 'glia endfoot'
+  * new 'perimeters' dataset
+* New metadata dataset containing provenance and versioning information:
+  * 'creator': The software used to create the morphology.
+  * 'software_version': major.minor.patch.sha1 of the software that wrote the
+    file
+  * 'creation_time': date & time when the file was written
+  * 'version': An attribute array of two U32LE integers that stores the major
+    and minor version numbers.
+
+## New dependencies
+
+None
+
+## API
+
+### Additions
+
+    SectionTypes
+    {
+        // after existing ones for neurons
+        SECTION_GLIA_PROCESS = 2,
+        SECTION_GLIA_ENDFOOT = 3,
+        // ...
+    };
+
+    MorphologyVersion
+    {
+        // between existing ones
+        MORPHOLOGY_VERSION_H5_1 = 1,
+        MORPHOLOGY_VERSION_H5_2 = 2,
+        MORPHOLOGY_VERSION_H5_1_1 = 3,
+        // ...
+    };
+
+    class Morphology
+        // Default parameter brion::MorphologyRepairStage for all functions to
+        // brion::MORPHOLOGY_UNDEFINED. Needed only for V2 and throws there if
+        // brion::MORPHOLOGY_UNDEFINED.
+
+### New
+
+    /** The cell family represented by brion::Morphology. */
+    CellFamily
+    {
+        FAMILY_NEURON = 0,
+        FAMILY_GLIA = 1
+    };
+
+    class Morphology
+    {
+        ...
+        /**
+         * @return the cell family of that morphology. FAMILY_GLIA only for
+         *         MORPHOLOGY_VERSION_H5_1_1 with a perimeter dataset,
+         *         otherwise FAMILY_NEURON.
```

Comment:
Not clear. I'd simply say "the cell family of this morphology". The rest is an implementation detail and I as I read is not fully precise. Neurons can also have a perimeter according to format 1.1, but it's optional in this case. On the other hand, the dataset is mandatory for glial cells.